assets/questions/rle.json

```diff
@@ -92,7 +94,7 @@
         "acquired_skills": ["String Manipulation", "Sets", "Arrays", "Maps"],
         "input_function": null,
         "output_function": null,
-        "main_function": "sample.DecodeEncodedString",
+        "main_function": "DecodeEncodedString",
```

Comment:
Sounds like Decode would be an output_function, then? Seems similar to my question a while back on how you'd evaluate whether a set of numbers that are output by a student has a mean of X -- here, you're evaluating whether a student's encoded output can be decoded.